lib/wasix/Cargo.toml

```diff
@@ -135,11 +193,17 @@ remote-vnet = ["virtual-net/remote"]
 
 logging = ["tracing/log"]
 disable-all-logging = ["tracing/release_max_level_off", "tracing/max_level_off"]
-enable-serde = ["typetag", "virtual-fs/enable-serde", "wasmer-wasix-types/enable-serde"]
+enable-serde = [
+    "typetag",
+    "virtual-fs/enable-serde",
+    "wasmer-wasix-types/enable-serde",
+]
 
 [package.metadata.docs.rs]
 features = [
-    "wasmer/sys", "webc_runner_rt_wcgi",
-    "webc_runner_rt_emscripten", "sys-default",
+    "wasmer/sys",
+    "webc_runner_rt_wcgi",
+    "webc_runner_rt_emscripten",
+    "sys-default",
```

Comment:
Yeah, this has been a bit of a problem.

I think some IDE does this (VS Code?), and it's always annoying in diffs.

Reply:
I'd argue that just because we don't link toml files shouldn't mean that formatting them with a formatter is a bad thing. Also, it'd be a huge pain for us VS Code folk. Somebody's gonna auto-format all of those tomls eventually.

Reply:
To disable auto-format for toml files, this has to go in the VS Code settings file:

```json
    "[toml]": {
        "editor.formatOnSave": false,
        "editor.formatOnPaste": false
    },
```

We may want to document this somewhere so other folk can discover it too. cc @dynamite-bud